### PR TITLE
Added Relief and FCBF as in issue #1126

### PR DESCRIPTION
**Pull request by shark-S**
Add Relief and FCBF according to issue #1126 in Select Relevant Features' preprocessor.

### CODE REVIEW

File: Orange/widgets/data/owpreprocess.py [hunk 1]
```diff
@@ -506,7 +508,9 @@ def __init__(self, parent=None):
         self.__uni_fs.setItems(
             [{"text": "Information gain", "tooltip": ""},
              {"text": "Gain ratio"},
-             {"text": "Gini index"}
+             {"text": "Gini index"},
+             {"text": "ReliefF"},
+             {"text": "Fast Correlation Based Filter"}
```
Comment (kernc) [inline Orange/widgets/data/owpreprocess.py]: Best just end this line with a comma as well so it won't need changing when the list is added to. Python doesn't complain about a trailing comma.

File: Orange/widgets/data/owpreprocess.py [hunk 2]
```diff
@@ -492,6 +492,8 @@ class FeatureSelectEditor(BaseEditor):
         ("Information Gain", preprocess.score.InfoGain),
         ("Gain ratio", preprocess.score.GainRatio),
         ("Gini index", preprocess.score.Gini),
+        ("ReliefF",preprocess.score.ReliefF),
+        ("Fast Correlation Based Filter",preprocess.score.FCBF)
```
Comment (kernc) [inline Orange/widgets/data/owpreprocess.py]: Space after comma makes it somewhat more readable.

**Comment by kernc** [comment]
Looks good, just some minor issues.

**Comment by shark-S** [comment]
Thanks @kernc  i will improve it.

**Comment by kernc** [comment]
@shark-S, this looks good. Could you just squash the two commits into one?

**Comment by shark-S** [comment]
thanks @kernc  . I will do it.

**Comment by kernc** [comment]
Thanks for this!

**Pull request merged**